packages/tab/src/mwc-tab-base.ts

```diff
@@ -66,24 +66,24 @@ export class TabBase extends BaseElement {
     this.mdcFoundation.handleClick();
   }
 
-  createRenderRoot() {
+  protected createRenderRoot() {
     return this.attachShadow({mode: 'open', delegatesFocus: true});
   }
 
-  connectedCallback() {
+  protected connectedCallback() {
```

Comment:
I still a tsc error here because this is `public` in `HTMLElement`. Remove `protected`.

I see a bunch of other ones too, e.g. with `activate` and `deactivate` -- try running `npm run build:typescript` again?

Reply:
Ouch! I left that change at unstaged files!

Reply:
But no errors about `activate` and `deactivate`...

Reply:
<img width="907" alt="Screenshot 2019-08-15 at 19 30 34" src="https://user-images.githubusercontent.com/1007051/63113631-36ae4880-bf93-11e9-8288-d17a738379c9.png">

Reply:
Try `npm install`? Looks like your build is totally broken right now.

Reply:
1. `rm -rf node_modules; npm install`
2. `npm run build`:

<img width="1143" alt="Screenshot 2019-08-15 at 19 40 37" src="https://user-images.githubusercontent.com/1007051/63114247-aec93e00-bf94-11e9-907b-56657f837a78.png">

Reply:
Do the steps at https://github.com/material-components/material-components-web-components#contributing-guide work? Try `npm run clean && npm install && npm run bootstrap && npm run build`.

Reply:
Now it works, thanks!

Reply:
<img width="1259" alt="Screenshot 2019-08-15 at 19 52 31" src="https://user-images.githubusercontent.com/1007051/63114870-41b6a800-bf96-11e9-9e05-3ac5881ea2b6.png">

Reply:
For the `mwc-drawer-base.ts` error, you want something like `this._previousFocus = (this.getRootNode() as ShadowRoot).activeElement as HTMLElement`.

For the `ripple-directive` error, something like `let rippleFoundation = part.value as MDCRippleFoundation | typeof noChange`, and then cast `as MDCRippleFoundation` below. (There might be something we could do to `noChange` so that TypeScript can figure out that the type must be `MDCRippleFoundation` after the check automatically, but for now I would just cast).